localisation/00_wc_sad.csv

```diff
@@ -708,3 +708,5 @@ nick_the_corruptor;the Corruptor;;[From.Getderdie] Verderber[From.GetinFemale];;
 text_effect_qiraji_retinue;Enables �GSilithid Swarm�! retinue that is normally available to �YQiraji�! culture only;;Aktiviert das �GSilithiden-Schwarm�!-Gefolge f�r die �YQiraji�!-Kultur;;;;;;;;;;;x
 text_effect_naga_retinue;Enables �GRoyal Guard�! retinue that is normally available to �YNaga�! culture only;;Aktiviert das �GK�nigswache�!-Gefolge f�r die �YNaga�!-Kultur;;;;;;;;;;;x
 text_effect_titanforged_retinue;Enables �GTitan-forged Legion�! retinue that is normally available to �YTitan-forged�! culture group only;;Aktiviert das �GTitanische Legion!-Gefolge f�r die �Ytitanische�! Kultur;;;;;;;;;;;x
+twilight_architecture;Twilight Architecture;;;;;;;;;;;x
+twilight_architecture_desc;This character uses powers of Old Gods to use elementium as building material.;;;;;;;;;;;x
```

Comment:
I'm not an expert, but shouldn't it be:
```suggestion
twilight_architecture_desc;This character uses the powers of the Old Gods to use elementium as a building material.;;;;;;;;;;;x
```

Reply:
Is some sort of magic required to use elementium in the first place?

Reply:
@arithon no, but there's Old Gods' background.
https://wow.gamepedia.com/Elementium

Reply:
I know of the background, but I think we should rephrase it a little then... How about: "This character uses elementium in conjunction with the powers of the Old Gods as a building material."
?

Reply:
Up to @sejtalk, I'm fine with both.

Reply:
@sejtalk the resolve reverted by 893403bd6757d4a38c71e8fe39b64b154bb993ec.

Reply:
i know i forgot to push ok